pkg/operator/operator.go

```diff
@@ -401,6 +412,39 @@ func New(
 	)
 	o.informerFactories = append(o.informerFactories, kubeInformersOperatorNS)
 
+	configInformers := configv1informers.NewSharedInformerFactory(configClient, 10*time.Minute)
+	missingVersion := "0.0.1-snapshot"
+	desiredVersion := missingVersion
+	if envVersion, exists := os.LookupEnv("RELEASE_VERSION"); exists {
+		desiredVersion = envVersion
+	}
+
+	// By default when the enabled/disabled list of featuregates changes, os.Exit is called.
+	featureGateAccessor := featuregates.NewFeatureGateAccess(
+		desiredVersion, missingVersion,
```

Comment:
`desiredVersion`  should be the same as the `version` input parameter?
```suggestion
		version, missingVersion,
```